app/human-detector-app/__tests__/BackendService.test.ts

```diff
@@ -0,0 +1,113 @@
+import * as BackendService from '../services/BackendService';
+import * as ServerConfig from '../config/ServerConfig';
+import User from '../classes/User';
+import Group from '../classes/Group';
+import Camera from '../classes/Camera';
+import Notification from '../classes/Notification';
+
+/**
+ * This file is to take care of tests on methods that will be used
+ * to reach our REST API backend endpoints.
+ */
+
+const nock = require('nock');
+
+// test putting notification key
+describe(BackendService.sendNotifyTokenAPI, () => {
+  it('should return code 200 for successful token sending', async () => {
+    const newUser: User = new User('tucker', '987654', true);
+    const scope = nock(ServerConfig.apiLink)
+      .put(ServerConfig.getSendNotifKeyUrlExtension(newUser.userID))
+      .reply(200, 'success');
+
+    const result: Promise<string> = await BackendService.sendNotifyTokenAPI(
```

Comment:
Since you do an await, shouldn't the result just be a string?

Reply:
whoopsies: 3580cc1ef9be65c183ba920c4657a04229740792